### ISSUE TITLE
getPackageInfo should not attempt to load classes

### ISSUE THREAD

**Issue by galderz** [oracle/graal#2111]
Master breaks one of quarkus test (see [quarkus#6685](https://github.com/quarkusio/quarkus/issues/6685)).

The relevant part is:
```
Caused by: com.oracle.svm.core.jdk.UnsupportedFeatureError: JDK11OrLater: Target_java_lang_ClassLoader.loadClass(Target_java_lang_Module, String)
	at com.oracle.svm.core.util.VMError.unsupportedFeature(VMError.java:101)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at java.lang.Package.getPackageInfo(Package.java:420)
	at java.lang.Package.getAnnotations(Package.java:467)
	at org.eclipse.yasson.internal.AnnotationIntrospector.collectAnnotations(AnnotationIntrospector.java:757)
```

One possible fix would be to add a substitution in `getPackageInfo` to avoid ending attempting to load classes. A PR follows up.

**Comment by eginez** [comment]
related to https://github.com/oracle/graal/issues/2093

**Comment by galderz** [comment]
Thanks @eginez! Sorry I missed the original issue ☺️

**Comment by eginez** [comment]
Fixed by 871f978
